.github/workflows/framework-tests.yaml

```diff
@@ -37,3 +37,36 @@ jobs:
 
       - name: Run unit tests
         run: tox -e unit
+
+  test-real-pebble:
+    runs-on: ${{ matrix.os }}
+
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        python-version: [3.5, 3.6, 3.7, 3.8, 3.9]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Set up Go 1.17
+        uses: actions/setup-go@v1
+        with:
+          go-version: 1.17
+
+      - name: Install tox
+        run: pip install tox
+
+      - name: Install Pebble
+        run: go install github.com/canonical/pebble/cmd/pebble@latest
+
+      - name: Start Pebble
+        run: umask 0 && PEBBLE=/tmp/pebble $HOME/go/bin/pebble run --create-dirs &
```

Comment:
```suggestion
        run: |
          umask 0
          pebble run --create-dirs &
        env:
          PEBBLE: /tmp/pebble
```

Reply:
I *think* that the `actions/setup-go` action adds $GOBIN to the $PATH automatically